log-elasticsearch/CHANGELOG.md

```diff
@@ -1,3 +1,6 @@
+# log-elasticsearch-0.10.3.0 (2020-08-10)
+* ElasticSearch 7.x compatibility
```

Comment:
Can we be a bit more specific?
We only get ES 7.x compatibility by using the ES 5 client and not doing anything that makes it unhappy.

Maybe:
> Add ElasticSearch 7.x compatibility by dropping explicit field mapping while still using `bloodhound` V5 client